core/Core.php

```diff
@@ -113,7 +113,12 @@
 }
 
 // Now that the class manifest is up, load the static configuration
-$configManifest = new SS_ConfigStaticManifest(BASE_PATH, false, $flush);
+if (defined('SS_CONFIGSTATICMANIFEST')) {
```

Comment:
@tractorcow Where are the files located that I need to modify for adding the documentation?